lib/grid.rb

```diff
@@ -2,11 +2,27 @@ class Grid
   attr_accessor :structure
 
   def initialize
-    @structure = grid_structure
+    @structure = build_grid
   end
 
-  def mark(coordinate)
-    structure[coordinate] = 'X'
+  def player_x
```

Comment:
I think before we talked about moving these markers here so that random objects wouldn't have to know about the marker. But now that you have players, it makes a lot of sense to me that they would know their own marker. If we wanted to have emoji characters for player marks instead of X and O, we could.

Reply:
✔️